robottelo/helpers.py

```diff
@@ -688,8 +688,7 @@ def host_provisioning_check(ip_addr):
     :return: ssh command return code and stdout
     """
     result = ssh.command(
-        u'for i in {{1..60}}; do ping -c1 {0} && exit 0; sleep 20;'
-        u' done; exit 1'.format(ip_addr)
+        'for i in {{1..60}}; do ping -c1 {0} && exit 0; sleep 20;' ' done; exit 1'.format(ip_addr)
```

Comment:
Any way to not merge split strings back into one line? Or to actually make them single string?
```suggestion
        'for i in {{1..60}}; do ping -c1 {0} && exit 0; sleep 20; done; exit 1'.format(ip_addr)
```

Reply:
black doesn't combine strings, but i can certainly make the change manually. nice catch!